src/main/resources/log4j2.xml

```diff
@@ -33,7 +33,7 @@
 		</RandomAccessFile>
 		<Lambda name="Lambda">
```

Comment:
Doesn't this Lambda block and any refs to it no longer work without the `aws-lambda-java-log4j2` dependency?

Reply:
I think we dont need this at all. CloudWatch logs will pick up the STDOUT console output be default. Will test and remove this if possible.

Reply:
@steven-sheehy yup, dont need this. Removed this completely